src/MySqlConnector/Core/ConnectionPool.cs

```diff
@@ -326,6 +359,14 @@ private ConnectionPool(ConnectionSettings cs)
 				cs.LoadBalance == MySqlLoadBalance.Random ? RandomLoadBalancer.Instance :
 				cs.LoadBalance == MySqlLoadBalance.FewestConnections ? new FewestConnectionsLoadBalancer(this) :
 				(ILoadBalancer) new RoundRobinLoadBalancer();
+
+			Id = Interlocked.Increment(ref s_poolId);
+			m_logArguments = new object[] { "Pool{0} ".FormatInvariant(Id) };
```

Comment:
I'd find the log statements more readable if you omitted the trailing space here and included them after `{0}` elsewhere.